src/containers/adminPage/SelectionPanel.jsx

```diff
@@ -56,7 +56,15 @@ function SelectionPanel(props) {
                                                 id={id}
                                                 className="entur-radio-checkbox--round"
                                                 checked={isChecked}
-                                                onChange={() => updateHiddenList(id, 'stops')}
+                                                onChange={() => {
+                                                    let stopUnchecked = false
+                                                    for (stop in stops){
```

Comment:
We try to avoid for loops, and want to keep everything as functional as possible. 
Please use [.some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) or [.every](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every)

Maybe something like this works:
```
const stopUnchecked = stops.some(stop => !onCheck(stop.id, 'stops'))
```